tvbwidgets/ui/spacetime_widget.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
```

Comment:
Could you pls include the header from the other files ?
You should also write a section about the `author` and put your name explicitly @peeplika